package.json

```diff
@@ -59,12 +67,20 @@
         "@babel/preset-react": "^7.12.13",
         "@babel/preset-typescript": "^7.13.0",
         "@babel/runtime": "^7.13.10",
+        "@storybook/addon-actions": "^6.1.21",
+        "@storybook/addon-essentials": "^6.1.21",
+        "@storybook/addon-links": "^6.1.21",
+        "@storybook/react": "^6.1.21",
+        "@testing-library/jest-dom": "^5.11.9",
+        "@testing-library/react": "^11.2.5",
         "@types/jest": "^26.0.20",
         "@typescript-eslint/eslint-plugin": "^4.19.0",
         "@typescript-eslint/parser": "^4.19.0",
         "ace-builds": "^1.4.12",
+        "addon-redux": "^1.1.0",
         "babel-loader": "^8.1.0",
         "copyfiles": "^2.0.0",
+        "core-js": "^3.9.1",
         "css-loader": "^5.2.0",
         "css-minimizer-webpack-plugin": "^1.3.0",
         "cssnano": "^4.1.11",
```

Comment:
You can remove cssnano as an explicit dependency, since that was there for the css minimizer.